Jenkinsfile.nightly

```diff
@@ -14,25 +21,29 @@ pipeline {
 				'''
 			}
 		}
-		stage('Run integration tests') {
+		stage('Run integration tests without syncing') {
 			steps {
-				sh 'npm test -- mocha:extensive:integration'
+				sh 'MAXIMUM_TRANSACTION=1000 SYNCING_DISABLED=true npm test -- mocha:extensive:integration'
```

Comment:
Better to move `MAXIMUM_TRANSACTION` to Jenkin job parameters. So its not hardcoded and can be changed runtime, when needed.